gas/config/loongarch-parse.y

```diff
@@ -100,6 +100,23 @@ my_getExpression (expressionS *ep, const char *str)
   return ret;
 }
 
+/* op is var name?  */
+static void
+emit_const_var (const char *op)
+{
+  expressionS ep;
+  if (end <= top)
+    as_fatal (_("expr too huge"));
+  my_getExpression(&ep, op);
+  if (ep.X_op != O_constant)
+    as_bad("illegal operand: %s", op);
+  top->value.X_op = O_constant;
+  top->value.X_add_number = ep.X_add_number;
+  top->type = BFD_RELOC_LARCH_SOP_PUSH_ABSOLUTE;
```

Comment:
Does this mean the resulting object file will have this reloc record? Because we may be working with lld/mold that cannot process stack-based relocations, in which case we actually want to substitute the value in, as if the user had written the concrete value in the respective slot. Otherwise the approach is mostly okay.

Reply:
Once it is recognized as a constant, it will discarded this reloc in subsequent processing, and obviously object file will have not this reloc record.